Makefile

```diff
@@ -9,6 +9,7 @@ GOVET_IGNORE_RULES = \
 # List of executables needed for 'make generate'
 GENERATE_DEPENDENCIES = \
 	github.com/golang/mock/mockgen \
+	github.com/gogo/protobuf/protoc-gen-gogoslick \
```

Comment:
Yup. I don't want to block this from hitting `x/`, but agreed with @akshayjshah and I'll help make sure this stuff gets OK'd before we introduce to any customers or take out of the experimental package.